modules/kubernetes-addons/kube-prometheus-stack/main.tf

```diff
@@ -13,3 +26,75 @@ resource "kubernetes_namespace_v1" "prometheus" {
     }
   }
 }
+
+resource "kubectl_manifest" "prometheus_crd_alertmanagerconfigs" {
+  yaml_body         = data.http.prometheus_crd_alertmanagerconfigs.response_body
+  server_side_apply = true
+}
```

Comment:
This seems like a good pattern but I am not if sure if this is a breaking change. What would happen to existing users using CSA?